crates/containerd-shim-wasm/Cargo.toml

```diff
@@ -28,14 +28,16 @@ serde_json = { workspace = true }
 tempfile = { workspace = true, optional = true }
 thiserror = { workspace = true }
 wat = { workspace = true }
-tokio = { workspace = true, features = ["rt-multi-thread"] }
+tokio = { workspace = true, features = ["rt-multi-thread", "macros"] }
 futures = { version = "0.3.30" }
 wasmparser = { version = "0.226.0" }
 tokio-stream = { version = "0.1" }
 sha256 = { workspace = true }
 serde_bytes = "0.11"
 prost = "0.13"
 toml = "0.8"
+trait-variant = "0.1"
+tokio-async-drop = "0.1"
```

Comment:
What is this crate used for?

Reply:
Good catch!
Initially I had added as a dev-dependency for the `sandbox::shim::local::tests`, but then thought we could actually use this for the `sandbox::containerd::client` for when we drop the lease.
I'll update the lease to use tokio_async_drop.